content/200-orm/300-prisma-migrate/300-workflows/10-seeding.mdx

```diff
@@ -385,6 +387,117 @@ psql file.sql
 
 </TabbedContent>
 
+### Seeding your database with <inlinecode>@snaplet/seed</inlinecode>
+
+[`@snaplet/seed`](https://docs.snaplet.dev/seed/getting-started/overview) offers a toolkit designed to understand your database schema,
+enabling you to generate production-accurate data and efficiently seed your database. It use the full power of TypeScript to provide you with
+a type-safe and auto-completed experience to write your seed scripts.
+
+It supports PostgreSQL and SQLite.
+
+1. Begin by setting up a local database using `npx prisma migrate dev`. For instance, consider the following Prisma schema featuring a `User` and `Post` model:
+
+   ```prisma file=schema.prisma
```

Comment:
it looks like this block is indented by 3. I've seen that before elsewhere in our docs, but am not sure why. Do things still look ok if this is indented by 2?

Reply:
Indeed I didn't noticed, I've actually copy/pasted this from the examples in the above recipes.

Turning it to 2 didn't changed the output.

Most of them uses 3 spaces indentations. Should I fix them all ? Or keep that for another PR ?